spec/rmagick/image_list/mosaic_spec.rb

```diff
@@ -9,9 +9,9 @@
 
   it "raises an error when images is not set" do
     image_list = described_class.new
-    list = image_list.copy
+    image_list = image_list.copy
```

Comment:
Maybe, this line is incorrect.
image_list variable Is already existed.

Reply:
hmm, It seems that there is no problem.

Reply:
Actually, @Watson1978 you found a mistake. I'll fix it.